.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## What is the goal of this pull request?
+
+## What does this pull request change?
+
+## What work remains to be done?
+
+## Does this pull request change the functionality of the code?
+
+YES | NO
+
+### How?
+
+## Do you consider it adequately tested?
+
+YES | NO
+
+## Does it change the deployment pipeline?
+
+YES | NO
+
+## Related Bugs
+
+## Related Pull Requests
```

Comment:
I think it's rare enough to mention another PR immediately when opening another PR, so I would also get rid of this question too.  I think if contributors wanted, they could add it in notes.